source/documentation/breaking-changes/bogus-combinators.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Breaking Change: Invalid Combinators'
+introduction: >
+  Sass has historically been very permissive about the use of leading, trailing,
+  and repeated combinators in selectors. These combinators are being deprecated
+  except where they're useful for nesting.
+---
+
+Sass has historically supported three invalid uses of combinators:
+
+- Leading combinators, as in `+ .error {color: red}`.
+
+- Trailing combinators, as in `.error + {color: red}`.
+
+- Repeated combinators, as in `div > > .error {color: red}`.
+
+None of these are valid CSS, and all of them will cause browsers to ignore the
+style rule in question. Supporting them added a substantial amount of complexity
+to Sass's implementation, and made it particularly difficult to fix various bugs
+related to the `@extend` rule. As such, we [made the decision] to remove support
+for these uses.
+
+[made the decision]: https://github.com/sass/sass/issues/3340
+
+**There is one major exception**: leading and trailing combinators may still be
+used for nesting purposes. For example, the following is still very much
+supported:
+
+<!-- prettier-ignore-start -->
+{% codeExample 1 %}
+.sidebar > {
+  .error {
+    color: red;
+  }
+}
+===
+.sidebar >
+  .error
+    color: red
+{% endcodeExample %}
+<!-- prettier-ignore-end -->
```

Comment:
Another issue with md is that the contents of these blocks are un-indented by prettier, so I had to add these comments to preserve whitespace.

I think we could somehow configure prettier to ignore these blocks, or extend the markdown parser to handle a custom syntax like [this](https://github.com/vokimon/markdown-customblocks#basic-usage).

Those both sound complex, so I wonder if we should just go back to `.liquid` whenever we use `{% codeExample %}` 😕

Reply:
Yeah... I'm reverting to my earlier "markdown is too magic" stance. I think we should use it sparingly, and only in cases where we've confirmed it doesn't break things. For a generated static-site in markup, it just feels to risky to rely heavily on a language that interprets indentation so meaningfully.

Reply:
Yea we ran into this early on when first implementing the `codeExample` logic, and since then my default has been to use .liquid templates whenever we have code examples on the page.

Reply:
Got it, back to .liquid then 👍